athenahealth/enum/appointment_status.go

```diff
@@ -0,0 +1,41 @@
+package enum
+
+import "strconv"
+
+// AppointmentStatus is derived from https://docs.athenahealth.com/api/api-ref/appointment#Book-appointment
+type AppointmentStatus string
+
+const (
+	Cancelled     AppointmentStatus = "x"
+	ChargeEntered AppointmentStatus = "4"
+	CheckedIn     AppointmentStatus = "2"
+	CheckedOut    AppointmentStatus = "3"
+	Future        AppointmentStatus = "f"
+	Open          AppointmentStatus = "o"
```

Comment:
Can you please prefix these with `AppointmentStatus` i.e. `AppointmentStatusOpen` to provide namespacing on these enums? This becomes especially important for when they're moved back into the top level `athenahealth` package